packages/common/src/store/ui/modals/hide-confirmation-modal/index.ts

```diff
@@ -0,0 +1,22 @@
+import { createModal } from '../createModal'
+
+export type HideConfirmationModalState = {
+  confirmCallback: () => void
+  cancelCallback?: () => void
+}
+
+const hideConfirmationModal = createModal<HideConfirmationModalState>({
```

Comment:
this name is super confusing

maybe `hideContentConfirmationModal` or something?

Reply:
nice, updated